rpcs3/rpcs3qt/game_list_frame.cpp

```diff
@@ -2057,12 +2058,12 @@ void game_list_frame::PopulateGameGrid(int maxCols, const QSize& image_size, con
 /**
 * Returns false if the game should be hidden because it doesn't match search term in toolbar.
 */
-bool game_list_frame::SearchMatchesApp(const std::string& name, const std::string& serial)
+bool game_list_frame::SearchMatchesApp(const QString& name, const QString& serial, const QString& custom_name)
 {
 	if (!m_search_text.isEmpty())
 	{
-		QString searchText = m_search_text.toLower();
-		return qstr(name).toLower().contains(searchText) || qstr(serial).toLower().contains(searchText);
+		const QString searchText = m_search_text.toLower();
+		return name.toLower().contains(searchText) || serial.toLower().contains(searchText) || custom_name.toLower().contains(searchText);
```

Comment:
In my opinion you should first check and match the custom name if it exists. Only match name if no custom name exists (false positives? But that is for discussion.)

Reply:
That's debatable though, but I guess the only case where it **truly** would matter are "meme" renames, like renaming "Game XYZ" to "bad gem".